store/storetest/post_store.go

```diff
@@ -526,7 +526,9 @@ func testPostStoreGetWithChildren(t *testing.T, ss store.Store) {
 		}
 	}
 
-	store.Must(ss.Post().Delete(o3.Id, model.GetMillis(), ""))
+	if err := ss.Post().Delete(o3.Id, model.GetMillis(), ""); err != nil {
```

Comment:
nit: I would prefer here:
```go
err := ss.Post().Delete(o3.Id, model.GetMillis(), "")
require.Nil(t, err)
```
But anyway, is still consistent with the rest of the code, so... up to you if you want to change it.